src/app/components/source/UserPrivacy.js

```diff
@@ -17,6 +18,30 @@ import { stringHelper } from '../../customHelpers';
 import { units } from '../../styles/js/shared';
 import globalStrings from '../../globalStrings';
 
+const useStyles = makeStyles({
+  linkStyle: {
+    textDecoration: 'underline',
+  },
+  headerStyle: {
+    margin: `${units(2)} 0`,
+    marginTop: units(6),
+  },
+  style: {
+    margin: `${units(2)} 0`,
+  },
+  cardStyle: {
+    margin: `${units(2)} 0`,
```

Comment:
Great work, thank you! Can I ask you to please also remove these `units` call and the import in favor of the `theme.spacing` usage?

Reply:
Sure, I'll fix this.